pkg/neorpc/result/invoke.go

```diff
@@ -99,25 +105,25 @@ func (r *Iterator) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
+	if iteratorAux.Value != nil {
+		r.Values = make([]stackitem.Item, len(iteratorAux.Value))
+		for j := range r.Values {
+			r.Values[j], err = stackitem.FromJSONWithTypes(iteratorAux.Value[j])
+			if err != nil {
+				return fmt.Errorf("failed to unmarshal iterator values: %w", err)
+			}
+		}
+	}
+	r.Truncated = iteratorAux.Truncated
 	if len(iteratorAux.Interface) != 0 {
 		if iteratorAux.Interface != iteratorInterfaceName {
 			return fmt.Errorf("unknown InteropInterface: %s", iteratorAux.Interface)
 		}
-		var iID uuid.UUID
-		iID, err = uuid.Parse(iteratorAux.ID)
+		iID, err := uuid.Parse(iteratorAux.ID)
```

Comment:
Useless change, revert it. I'm asking for this because I appreciate preserving git history (`git blame` will tell you that this line was changed in this PR although this change is not expected to happen during session expansion development).